draco/synthesis/mockcatalog.py

```diff
@@ -1079,7 +1071,7 @@ def elg_velocity_error(z, rng):
         percentiles:
             "Additionally, we can assess with repeats that 99.5, 95, and 50
             percent of our redshift estimates have a precision better than
-            300 km s−1, 100 km s−1, and 20 km s−1, respectively."
```

Comment:
Why is this converting minus signs to dashes?  Is unicode not allowed in docstrings?

If the − really isn't wanted, it might be better to just write "km/s" here.